components/elm/src/data_types/TopounitType.F90

```diff
@@ -45,14 +46,15 @@ module TopounitType
     ! this is for efficiency, since most loops will go over t in the outer loop, and
     ! landunit type in the inner loop)
     integer , pointer :: landunit_indices (:,:) => null() 
+	logical , pointer :: active       (:) => null() ! true=>do computations on this topounit 
```

Comment:
Fix indentation

Reply:
@thorntonpe I have fixed indentation in my latest commit. Thanks.